gremlin/templates/gremlin-service-account.yaml

```diff
@@ -9,6 +9,28 @@ metadata:
     {{- with .Values.gremlin.serviceAccount.annotations }}
       {{- toYaml . | nindent 4 }}
     {{- end }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: gremlin-daemon-watcher
+rules:
+  - apiGroups: [""]
```

Comment:
Q: does this mean apiGroups is NULL or apiGroups wildcard?

Reply:
Ah, an empty string means the core API group. 
Also, if it's helpful, the error we see when we do not have the correct permission is:

```
User "system:serviceaccount:gremlin:gremlin" cannot get resource "nodes" in API group "" at the cluster scope: Forbidden.
```

Reply:
I believe `apiGroups: ["*"]` would denote a wildcard.